bull-transformer-generator/transformer-generator-core/src/test/java/com/hotels/beans/generator/core/mapping/MappingCodeFactoryTest.java

```diff
@@ -48,32 +49,30 @@ public void shouldReturnAnInstanceForMutableDestination() {
     }
 
     /**
-     * TODO remove this test after implementation of immutable mapping code.
+     * TODO: remove this test after implementation of immutable mapping code.
      */
     @Test(expectedExceptions = NotImplementedException.class)
     public void shouldFailForImmutableDestination() {
         underTest.of(Source.class, ImmutableDestination.class);
     }
 
     /**
-     * TODO remove this test after implementation of mixed mapping code.
+     * TODO: remove this test after implementation of mixed mapping code.
      */
     @Test(expectedExceptions = NotImplementedException.class)
     public void shouldFailForMixedDestination() {
         underTest.of(Source.class, MixedDestination.class);
     }
 
-    @Test(expectedExceptions = AssertionError.class,
-            expectedExceptionsMessageRegExp = ".*UNSUPPORTED.*")
+    @Test(expectedExceptions = AssertionError.class, expectedExceptionsMessageRegExp = ".*UNSUPPORTED.*")
     public void shouldFailForUnsupportedDestinationType() {
         // GIVEN
         ClassUtils classUtils = given(mock(ClassUtils.class).getClassType(any()))
-                .willReturn(ClassType.UNSUPPORTED)
+                .willReturn(UNSUPPORTED)
                 .getMock();
-        MappingCodeFactory underTest = MappingCodeFactory.getInstance(classUtils);
 
         // WHEN
-        underTest.of(Source.class, Destination.class);
+        MappingCodeFactory.getInstance(classUtils).of(Source.class, Destination.class);
```

Comment:
Fabio, I think the original version followed the style of the other tests. This line looks a little less readable to me.

Reply:
ok

Reply:
reverted